glib/tests/structured_log.rs

```diff
@@ -1,3 +1,6 @@
+#![allow(clippy::unnecessary_to_owned)]
```

Comment:
This one seems surprising.

Reply:
The test is explicitly testing that things work with a `String` as well as with a `&str`. So technically the `to_owned()` is unnecessary indeed but that would make the test useless.